db/db_impl.cc

```diff
@@ -287,6 +287,8 @@ Status DBImpl::CloseImpl() {
       env_->UnSchedule(this, Env::Priority::BOTTOM);
   int compactions_unscheduled = env_->UnSchedule(this, Env::Priority::LOW);
   int flushes_unscheduled = env_->UnSchedule(this, Env::Priority::HIGH);
+  Status ret;
```

Comment:
This is going to be confusing in future. How about giving "Status s" a more limited scope?